troubleshooting/Ingestion/PostTelemetry.ps1

```diff
@@ -1,13 +1,29 @@
-$url="https://{IngestionEndpoint-from-Component-ConnectionString}/v2/track"
-$iKey="{test-iKey}"
-Write-Host $url, $iKey
+param (
+    [Parameter(Mandatory=$true)][string]$ConnectionString
+)
 
-$time = Get-Date
-$time = $time.ToUniversalTime().ToString('o')
+function ParseConnectionString {
+	param (
+		[string]$ConnectionString
+	)
 
-$availabilityData = @"
+	$Map = @{}
+
+	foreach ($Part in $ConnectionString.Split(";")) {
+		$KeyValue = $Part.Split("=")
+		$Map.Add($KeyValue[0], $KeyValue[1])
+	}
+
+	return $Map
+}
+
+$Map = ParseConnectionString($ConnectionString)
+$Url = $Map["IngestionEndpoint"] + "v2/track"
```

Comment:
IngestionEndpoint is an optional field in the ConnectionString.
Can you change this to default to "https://dc.services.visualstudio.com/"  ?

Reply:
Note that the InstrumentationKey is a required field. It's okay to fail if this is not provided.